### ISSUE TITLE
jest/no-identical-title doesn't report if using backticks

### ISSUE THREAD

**Issue by himynameisdave** [jest-community/eslint-plugin-jest#232]
Just checked and it appears that `jest/no-identical-title` doesn't report an error when specifying the title using backticks (even when not using string interpolation).

**Error:**
```js
describe('an odd case where identical titles are not reported', () => {
    it('this works as expected', () => {
        //  ...test code here
    });
    it('this works as expected', () => {
        //  ...test code here
    });
});
```

**Not reported:**
```js
describe('an odd case where identical titles are not reported', () => {
    it(`this does not work as expected`, () => {
        //  ...test code here
    });
    it(`this does not work as expected`, () => {
        //  ...test code here
    });
});
```

**Comment by macklinu** [comment]
Thank you for the report. It looks like we should also be checking that the first argument is a `TemplateLiteral` without any expressions (e.g. no `${}`) in the following area.

https://github.com/jest-community/eslint-plugin-jest/blob/3a1dc2dbcd39e5f92803e27882e1b4d6ee7e24b9/rules/no-identical-title.js#L54-L58

A PR is welcome - may take a further look at it this weekend, and if so, will comment that I'm picking it up. If you or anyone else would like to submit a PR with a fix, please comment!

**Comment by himynameisdave** [comment]
@macklinu I'd love to, thanks for pointing me to it!
